SConstruct

```diff
@@ -489,29 +489,6 @@ angle_sources = [
     "third_party/zlib/trees.c",
     "third_party/zlib/uncompr.c",
     "third_party/zlib/zutil.c",
```

Comment:
Is there any risk of similar symbol clashes with zlib?

Reply:
No, it should not conflict. ANGLE zlib is modified to use custom `_Cr_z` prefix (and seems to have other ANGLE specific changes).